hdeps/cli.py

```diff
@@ -84,6 +85,21 @@ def _stats_thread() -> None:
     is_flag=True,
     help="Output a theoretical install order instead of a tree",
 )
+@click.option(
+    "--print-legend",
+    is_flag=True,
+    help="Output the meaning of colors in a header",
+)
+@click.option(
+    "--force-color",
+    is_flag=True,
+    help="Force color on even if stdout is not a tty.  You can also set the FORCE_COLOR env var non-empty.",
+)
+@click.option(
+    "--no-color",
```

Comment:
Minor: you can simplify away the `--force-color` I think.

With click, you can do:
```
@click.option("--color/--no-color", default=None, is_flag=True, help="Use colors (on by default on ttys, you can also set the NO_COLOR env...)")
```

Then in code, compute default (and lookup env var) only when `color is None`)